arex-instrumentation-foundation/src/main/java/io/arex/foundation/serializer/JacksonSerializer.java

```diff
@@ -177,7 +180,9 @@ public List<BeanPropertyWriter> changeProperties(SerializationConfig config,
             if (MYBATIS_PLUS_CLASS_LIST.contains(className)) {
                 beanProperties.removeIf(beanPropertyWriter -> !StringUtils.equals(beanPropertyWriter.getName(), "paramNameValuePairs"));
             }
-
+            if (TK_MYBATIS_PLUS_CLASS_LIST.contains(className)){
+                beanProperties.removeIf(beanPropertyWriter -> beanPropertyWriter.getName().equals("table"));
```

Comment:
Recommend flipping the code, avoid NPE

 `("table").equals(beanPropertyWriter.getName())`